test/integration/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -635,14 +634,23 @@ var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure
 				}, util.Timeout, util.Interval).Should(gomega.Succeed())
 			})
 
+			ginkgo.By("Checking if an AdmissionCheck is Ready", func() {
```

Comment:
Is this enough? or should we also check that the Workload is Admitted?
I guess they might happen in the same API update.

Reply:
I don't think it matters whether it's Admitted or not in this scenario